prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="http://127.0.0.1:5500/contact-us/contact-us.html" target="_blank">Click here to go to the contact page</a>
```

Comment:
Hi Norman, this one should be just `href="contact-us/contact-us.html"`

Reply:
Resolved